api/config/genesis.go

```diff
@@ -75,9 +74,9 @@ func DefaultGenesisConfig() *GenesisConfig {
 
 	// we default to 10^5 SMH per account which is 10^17 smidge
 	g.InitialAccounts = map[string]GenesisAccount{
-		"0x1":       {Balance: big.NewInt(int64(math.Pow10(17))), Nonce: 0},
-		Account1Pub: {Balance: big.NewInt(int64(math.Pow10(17))), Nonce: 0},
-		Account2Pub: {Balance: big.NewInt(int64(math.Pow10(17))), Nonce: 0},
+		"0x1":       {Balance: 100000000000000000, Nonce: 0},
```

Comment:
It's now a little less clear to the naked eye what this value is. Could you add a comment? Something like, "Genesis balance is 10^17"